pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -767,11 +781,6 @@
       ^-  (unit (unit cage))
       (rof ~ care [our desk da+now] path)
     ::
-    ++  error-response
-      |=  [status=@ud =tape]
-      ^-  (quip move server-state)
-      %^  return-static-data-on-duct  status  'text/html'
-      (error-page status authenticated url.request tape)
```

Comment:
You might want to just leave this here, and have it call the outer one, so the callsites within this arm remain cleaner.

Alternatively, arguably even better, you could leave this as-is, and wrap just the `+handle-name` callsite in `return-static-data-on-duct`, and include the appropriate `error-page` calls there.

Main thing is just that it's here rn as a helper for this `|^`,  moving it out doesn't seem worth the sample-complexity cost to me. And the newly added callsite isn't particularly involved.

Reply:
Yeah you're right, my refactor here was a bad idea. Reverted to include `+error-response` inside `+handle-name`.